### PR TITLE
f-checkout@v0.29.0 - Add test helper file and checkout method tests

### PR DESCRIPTION
**Pull request by FayeCarter**
### Added
- `setup` test helper file.
- Tests for `Checkout` component methods.

### CODE REVIEW

File: packages/f-checkout/src/components/tests/Checkout.test.js [hunk 1]
```diff
@@ -329,216 +264,8 @@ describe('Checkout', () => {
         });
     });
 
-    describe('when form submitted', () => {
```
Comment (FayeCarter) [inline packages/f-checkout/src/components/tests/Checkout.test.js:332]: These tests have been moved into methods :: submitCheckout to match other tests

File: packages/f-checkout/src/components/tests/Checkout.test.js [hunk 2]
```diff
@@ -618,5 +345,296 @@ describe('Checkout', () => {
                 });
             });
         });
+
+        describe('submitCheckout ::', () => {
```
Comment (FayeCarter) [inline packages/f-checkout/src/components/tests/Checkout.test.js:273]: submitCheckout tests are not new, just moved to tidy up the file

File: packages/f-checkout/src/components/tests/Checkout.test.js [hunk 3]
```diff
@@ -618,5 +345,330 @@ describe('Checkout', () => {
...
+                it('should create validations for address', () => {
+                    // Assert
+                    expect(wrapper.vm.$v.fulfilment.address.line1).toBeDefined();
+                    expect(wrapper.vm.$v.fulfilment.address.city).toBeDefined();
+                    expect(wrapper.vm.$v.fulfilment.address.postcode).toBeDefined();
+                });
+            });
+        });
+
+        describe('handleErrorState ::', () => {
```
Comment (FayeCarter) [inline packages/f-checkout/src/components/tests/Checkout.test.js:547]: new tests start here :)

**Review by SachaZvetelman — APPROVED** [review]

**Review by DamianMullins — APPROVED** [review]

**Review by zlatinivanov-je — COMMENTED** [review]
Had a quick look, could not see anything wrong. Unless everyone else is happy with the naming `../helpers/setup.js` I was going to suggest something like `mocks` or `testData` or similar to identify it as relevant to the tests only.

**Pull request merged**